### DISCUSSION TITLE
Failed to create template under "play with docker" (and docker desktop for windows)

### DISCUSSION THREAD

**Discussion by KongNan** [coder/coder#12184] [General]
I followed the [doc from coder.com](https://coder.com/docs/v2/latest/install/docker#docker-compose 
) to try to install via docker compose with **root**, but get an error: permission denied.

How can I deal with this issue? Thanks! 

---------------------------
Here are the error messages for client and webui:

# console:
[node1] (local) root@192.168.0.18 ~/coder$ whoami
**root**

##step 2 clone
[node1] (local) root@192.168.0.18 ~/coder$ git clone https://github.com/coder/coder.git
Cloning into 'coder'...
Updating files: 100% (7087/7087), done.

##step 3 up
[node1] (local) root@192.168.0.18 ~/coder$ cd coder                                                                 
[node1] (local) root@192.168.0.18 ~/coder/coder$ docker compose up
[+] Running 20/20 layers ...

##step 4 login & check health: all ✔
##step 5 use a starter template "docker" ...
coder-coder-1     | 2024-02-16 02:10:37.524 [info]  provisionerd-83ea99bf345e-1.runner: dry-run provision failure  job_id=41aee34e-b418-49bd-9e8b-1449213afa15  error="**terraform plan: exit status 1**"
coder-coder-1     | 2024-02-16 02:10:37.531 [info]  terraform: recv done on Session  session_id=8e6fe6fd-ea86-41c6-931e-72334e4be1dd  error=EOF

# webui:  
10:30:28.256  （UTC+8)
Error: Error pinging Docker server: **Got permission denied while trying to connect to the Docker daemon socket** at unix:///var/run/docker.sock: Get "http://%2Fvar%2Frun%2Fdocker.sock/_ping": dial unix /var/run/docker.sock: connect: permission denied
on main.tf line 19, in provider "docker":
  19: provider "docker" {

**Comment by matifali** [comment]
When using via `docker-compose`, the `coder` user should be a member of the `docker` group.
Please see [this](https://coder.com/docs/v2/latest/install/docker#permission-denied-while-trying-to-connect-to-the-docker-daemon-socket) for more details.

**Reply by KongNan** [reply]
Thank you for your reply,  it's said that "See Docker's official documentation to [Manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user)",  but i'm already root... Let me have a try , thank you again!

**Reply by matifali** [reply]
The issue is not your user but the `coder` user within the container should be a member of the `docker` group. 
Also are you running the [WSL backend](https://docs.docker.com/desktop/wsl/)? You need WSL backend.
